packages/genesys-spark-components/src/components/stable/gux-modal/gux-modal.scss

```diff
@@ -49,12 +49,12 @@ slot[name='start-align-buttons']::slotted(:not(button, gux-button-slot)) {
       }
 
       &.gux-medium {
-        width: var(--gse-ui-modal-medium-width);
+        max-width: var(--gse-ui-modal-medium-width);
```

Comment:
I dont think this will work as if you  have a `medium` or `large` modal the width of the modal will be based on the content provided. I think maybe applying `display:flex` to the dialog element may work.